bin/Powheg/production/2017/13TeV/Higgs/MSSM/ggA_MuMu_NNPDF31_13TeV/gg_A_MSSM_h_5f_13TeV.input

```diff
@@ -0,0 +1,103 @@
+!hfact    %hfact    ! (default no dumping factor) dump factor for high-pt radiation: > 0 dumpfac=h**2/(pt2+h**2)
+
+
+
+!!!!!!!!!!!!  generator parameters     !!!!!!!!!!!!!!!
+! Parameters to allow or not the use of stored data
+use-old-grid    1 ! if 1 use old grid if file pwggrids.dat is present (<> 1 regenerate)
+use-old-ubound  1 ! if 1 use norm of upper bounding function stored in pwgubound.dat, if present; <> 1 regenerate
+! grids preparation
+ncall1 50000   ! number of calls for initializing the integration grid
+itmx1  5    ! number of iterations for initializing the integration grid
+ncall2 50000    ! number of calls for computing the integral and finding upper bound
+itmx2  5     ! number of iterations for computing the integral and finding upper bound
+foldcsi   1    ! number of folds on csi integration
+foldy     1    ! number of folds on  y  integration
+foldphi   1    ! number of folds on phi integration
+nubound 50000  ! number of bbarra calls to setup norm of upper bounding function
+icsimax  1     ! <= 100, number of csi subdivision when computing the upper bounds
+iymax    1     ! <= 100, number of y subdivision when computing the upper bounds
+xupbound 2d0   ! increase upper bound for radiation generation
+
+!ptsupp     0d0   ! (default 0d0)  mass param for Born suppression factor (generation cut) If < 0 suppfact = 1
+!bornonly   0      ! (default 0) if 1 do Born only
+!smartsig   0      ! (default 1) remember equal amplitudes (0 do not remember)
+!withsubtr  0      ! (default 1) subtract real counterterms (0 do not subtract)
+!withdamp    1      ! (default 0, do not use) use Born-zero damping factor
+testplots  1      ! (default 0, do not) do NLO and PWHG distributions
+!testsuda  1       ! (default 0, do not test) test Sudakov form factor
+!radregion 1       ! (default all regions) only generate radiation in the selected singular region  
+iseed    SEED    ! initialize random number sequence 
+!rand1    32      ! initialize random number sequence 
+!rand2    0      ! initialize random number sequence 
+!iupperisr 1 ! (default 1) choice of ISR upper bounding functional form 
+!iupperfsr 2 ! (default 2) choice of FSR upper bounding functional form 
+
+
+
+!!!!!!!!!!!! physical parameters  !!!!!!!!!!!!!!!
+gfermi 0.116637D-04        ! GF
+
+!!!!!!!!!!! MSSM parameter !!!!!!!!!
+scheme 2      ! Mass renormalization scheme. 2 = OS/Feynhiggs, 1 = DRBAR
+
+!!!!!!!!!!! MSSM OS parameter  !!!!!!!!!
+FHslhaflag 0  ! relevant when scheme=2 . If 0 read FeynHiggs input param from powheg-fh.in, otherwise from powheg-fh.slha
+FHdecaywidth 1 ! 1 = use the decay width provided by FeynHiggs
+! bottommass 4.75d0 ! If not set, the bottom OS mass is calculated with a 1-loop scheme change formula.
+
+
+!!!!!!!!!!! MSSM DRbar parameter  !!!!!!!!!
+mssmmbatmh 0 ! 1 = mb(mh) is used (from mssm-param-mh.slha), 0 = mb(M_susy) is used
+pdfreweight 1       ! PDF reweighting
+storeinfo_rwgt 1    ! store weight information
```

Comment:
Please remove. Will be automatically handled.

Reply:
Done

Reply:
reminder